clayui.com/plugins/gatsby-remark-use-clipboard/index.js

```diff
@@ -5,14 +5,39 @@
 
 const visit = require('unist-util-visit');
 
+const parseOptions = (meta) => {
+	const availableOptions = {};
+
+	if (meta) {
+		meta.map((opt) => {
```

Comment:
I believe that we don't want to abbreviate variables like these, `opt` should be `option`. @wincent correct me if I'm wrong

Reply:
Yeah, in general nope. Rather than answering this in lots of places, I created this PR to liferay-frontend-guidelines: https://github.com/liferay/liferay-frontend-guidelines/pull/195

Reply:
Changed `opt` to `option` let me know if there's anything else.